src/detection/fetch/repository/query.js

```diff
@@ -0,0 +1,21 @@
+import { gql } from '@apollo/client'
+
+export const GET_DETECTIONS = gql`
```

Comment:
Let's rename this query to `GET_DETECTION`.

Reply:
done